src/test/tests/appsConfig.test.ts

```diff
@@ -0,0 +1,120 @@
+import { assert } from 'chai';
+import * as path from 'path';
+import { ScanType } from '../../main/scanLogic/scanRunners/analyzerModels';
+import { ExcludeScannerName, Module, Scanner } from '../../main/types/jfrogAppsConfigSchema';
+import { AppsConfigModule, JFrogAppsConfig } from '../../main/utils/jfrogAppsConfig/jfrogAppsConfig';
+
+describe('JFrog Apps Config Tests', () => {
+    const jfrogAppsConfigDir: string = path.join(__dirname, '..', 'resources', 'jfrogAppsConfig');
+
+    it('Load full config', () => {
```

Comment:
We can break the test into two separate tests, each with its own particular assertion. for example, a check sast scanner can be encapsulated in its own it().

It will be clearer what fails in the config and it will enable us to scale if we would like to add additional specific module tests

Reply:
The bond between the tests is too strong to split them into subtests - it reads one file and parses it. This is the result of the parsing. If we add more tests, it will require creating another input yaml and by that a new it.

I do agree that the test method was too long and therefore I shorted the test.

Reply:
Maybe we can move the file read logic into the `before` section, each `it` will check a specific configuration section. Adding a new section to the config file in the future will make us add a single `it` test case.